src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <main class="main">
+      <div class="stars stars--0">
```

Comment:
These HTML elements, following semantic tags should be rather instead of `divs`:

```
<ul>
  <li></li>
  <li></li>
  ...
</ul>
```